portal-user-datastore-rdbms/src/main/java/com/telecominfraproject/wlan/portaluser/datastore/rdbms/PortalUserDAO.java

```diff
@@ -460,4 +465,25 @@ public PortalUser getByUsernameOrNull(int customerId, String username) {
             return null;
         }
 	}
+	
+	public List<Integer> getCustomerIdsForUsername(String username) {
+        LOG.debug("Looking up customerIds for username {} {}", username);
+
+        try {
+        	List<PortalUser> ret = this.jdbcTemplate.query(SQL_GET_BY_USERNAME,
+                    portalUserRowMapper, username);
+            
+            LOG.debug("Found List of Portal Users {}", ret);
+            
+            List<Integer> listOfCustomerIds = new ArrayList<>();
+            for (PortalUser portalUser : ret) {
+            	listOfCustomerIds.add(portalUser.getCustomerId());
+            }
+            
+            return listOfCustomerIds;
+        }catch (EmptyResultDataAccessException e) {
```

Comment:
This will not happen. JdbcTemplate.query will return an empty list if there is no data, it will not throw that exception

Reply:
Removed the catch, just added a debug log on the number of elements returned instead to verify when it does return empty.